source/src/molecular-unfolding/cdk/construct-dashboard.ts

```diff
@@ -27,14 +27,7 @@ export class Dashboard extends Construct {
     super(scope, id);
     this.props = props;
 
-    const defaultQuicksightUser = this.node.tryGetContext('quicksight_user') || process.env.QUICKSIGHT_USER;
-    const quickSightUserParam = new CfnParameter(this, 'quickSightUser', {
-      type: 'String',
-      default: defaultQuicksightUser,
-      description: 'Quicksight User',
-    });
-    const quicksightUser = `arn:aws:quicksight:us-east-1:${this.props.account}:user/default/${quickSightUserParam.valueAsString}`;
-
+    const quicksightUser = `arn:aws:quicksight:us-east-1:${this.props.account}:user/default/${this.props.quicksightUser}`;
```

Comment:
quicksight is available in multiple regions, is the user always in `us-east-1`?

Reply:
if you enable quicksight in your account,  the same user is used in all the regions. if you want to manage `Security & permissions` of the user, you need switch to `us-east-1`.